### PR TITLE
Add, remove tasks for To-do-list App

### PR DESCRIPTION
**Pull request by Kaweesi-Matia**
## below is the description for what is included in the to-do-list App for adding and removing tasks

- [ ] Add a function in index.js for adding a new task
- [ ] Implement a function for removing a task
- [ ] implement a function for editing task descriptions
- [ ] Add local storage to store to-do-list App changes

### CODE REVIEW

File: src/module/addRemoveEdit.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+};
+todoz.forEach(addTask);
+const modifyToDoList = () => {
+  const editInput = document.querySelectorAll('.newInput');
+  editInput.forEach((edits, indexy) => {
+    edits.addEventListener('change', () => {
+      todoz.forEach((todo, index) => {
+        if (indexy === index) {
+          todo.Description = edits.value;
+          localStorage.setItem('todoz', JSON.stringify(todoz));
```
Comment (ghost) [inline src/module/addRemoveEdit.js:58]: - Good work implementing the `add` and `remove` functionality. However, I would recommend that when editing a task, the changes should be saved in the local storage. Kindly implement that. This ensures that changes are saved even if the page is reloaded.

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @Kaweesi-Matia , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

YesYouAreGIF (2)

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Good PR Summary👍 
✅ You can add and remove a task👍 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/module/addRemoveEdit.js (hunk 1 above)
Comment (Kaweesi-Matia) [reply src/module/addRemoveEdit.js:58]: add local storage

**Review by mgmediaweb — APPROVED** [review]
Hi @Kaweesi-Matia , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved4

## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**